Unable to install linuxheaders on AMD system

**Describe the bug**

In my efforts trying to figure out a solution to #7483, I decided to try building mesa from source. However, this requires `linuxheaders` to be installed, which fails on my machine.

The first failure happens because when running `crew install -s mesa`, it tries to install `linuxheaders` before installing its build dependencies make, binutils, etc which predictably results in an error. Running `crew install linuxheaders` resolves the dependencies properly which gets me further, but the initial issue seems like a dependency resolution bug worth reporting upstream.

Once I've got all the dependencies, `linuxheaders` still fails to install with the error output below, complaining about a missing header file. I tried installing `linux_sources` to see if that would help, but no dice.

**To Reproduce**
Steps to reproduce the behavior:

1. I executed this command:

      ```shell
     crew install linuxheaders
      ```

Output:

```
linuxheaders: Linux headers for Chrome OS.
https://kernel.org/
Version: 5.4
License: GPL-2
Performing pre-flight checks...
No precompiled binary available for your platform, downloading source...
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint:   git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint:   git branch -m <name>
Initialized empty Git repository in /usr/local/tmp/crew/linuxheaders.20221116165841.dir/.git/
remote: Finding sources: 100% (72654/72654)
remote: Total 72654 (delta 4426), reused 20550 (delta 4426)
Receiving objects: 100% (72654/72654), 195.91 MiB | 5.41 MiB/s, done.
Resolving deltas: 100% (4426/4426), done.
From https://chromium.googlesource.com/chromiumos/third_party/kernel
 * branch                chromeos-5.4 -> FETCH_HEAD
 * [new branch]          chromeos-5.4 -> origin/chromeos-5.4
Updating files: 100% (68455/68455), done.
HEAD is now at f901a671e UPSTREAM: usb: xhci-mtk: relax TT periodic bandwidth allocation
Repository downloaded.
Building from source, this may take a while...
Preconfiguring package...
  HOSTCC  scripts/basic/fixdep
  HOSTCC  scripts/kconfig/conf.o
In file included from /usr/local/include/bits/posix1_lim.h:161,
                 from /usr/local/include/limits.h:195,
                 from scripts/kconfig/conf.c:7:
/usr/local/include/bits/local_lim.h:38:10: fatal error: linux/limits.h: No such file or directory
   38 | #include <linux/limits.h>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [scripts/Makefile.host:124: scripts/kconfig/conf.o] Error 1
make: *** [Makefile:590: defconfig] Error 2
/usr/local/lib/crew/lib/package.rb:279:in `system': Command failed with exit 2: bash
linuxheaders failed to install: `CREW_DISABLE_ENV_OPTIONS="1" bash -c PATH=/usr/local/tmp/crew/linuxheaders.20221116165841.dir/crew_bin:$PATH make defconfig` exited with 2
```

**Expected behavior**\

For `linuxheaders` to install successfully.


**Environment: Paste the output of `crew sysinfo -v` between the two `***` below**

***
<details><summary>Expand</summary>

- Architecture: `x86_64` (`x86_64`)
- Processor vendor: `AuthenticAMD`
- User space: `64-bit`
- Chromebrew Kernel version: `5.4`
- Chromebrew Running in Container: `false`

- Chromebrew version: `1.28.3`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib64`

- Last update in local repository: 665b38f `X11 updates, and updates of packages which used to depend upon xorg_lib (#7634) (29 hours ago)`

- OS variant: `Chrome OS`
- OS version: `zork-release/R107-15117.112.0`
- OS channel: `stable-channel`

</details>
***

**Hardware**: Asus Chromebook Flip CM5 (Ryzen 3250U)

Thanks in advance, and thanks to the Chromebrew team for all your work bringing the flexibility of real Linux to ChromeOS devices!

--- a/packages/linuxheaders.rb
+++ b/packages/linuxheaders.rb
@@ -40,6 +40,17 @@ class Linuxheaders < Package
        armv7l: 'cc9227b5b3af3abdaba4c1eb6b5df066af6c5f20629cee3a4e351b3e40521760',
        x86_64: 'ea112d55cb33823a473791d146d124ae5d2cce7fb72ebe0b047d5937957ee994'
     })
+  when '5.4'
+    binary_url({
+      aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/linuxheaders/5.4_armv7l/linuxheaders-5.4-chromeos-armv7l.tar.zst',
+       armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/linuxheaders/5.4_armv7l/linuxheaders-5.4-chromeos-armv7l.tar.zst',
+       x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/linuxheaders/5.4_x86_64/linuxheaders-5.4-chromeos-x86_64.tar.zst'
+    })
+    binary_sha256({
+      aarch64: 'b790edcfe5bcad8fc7e7b873a6ba8f9ad147b00849db2afead5f51e19f699df7',
+       armv7l: 'b790edcfe5bcad8fc7e7b873a6ba8f9ad147b00849db2afead5f51e19f699df7',
+       x86_64: 'e6ea539149f4333518787eb1e0fa436d0914990fd2899011c277ab418a9b78ba'
+    })
   when '5.10'
     binary_url({
       aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/linuxheaders/5.10_armv7l/linuxheaders-5.10-chromeos-armv7l.tar.zst',

